spec/game_spec.rb

```diff
@@ -2,12 +2,20 @@
 
 RSpec.describe Game do
   before(:each) do
-    @game = Game.new
+    @game1 = Game.create_games('./data/games.csv')[0]
   end
```

Comment:
I would imagine that we will need multiple game instances, so it might be better to put
```
games = Game.create_games("./data/games.csv")
@game1 = games[0]
@game2 = games[1]
...etc
```
But this will work for now, we can always refactor later

Reply:
Good catch. Pushed new update for this